app/music.rb

```diff
@@ -0,0 +1,12 @@
+class MusicAlbum
+  attr_accessor :on_spotify
+
+  def initialize(on_spotify)
+    @on_spotify = on_spotify
+  end
+
+  def can_be_archived?
+    true if super() && @on_spotify == true
+    false
+  end
+end
```

Comment:
MusicAlbum has an inheritance relationship with Item class please include those.

Reply:
Create Genre class with an association to the Item class (in a separate .rb file).
MusicAlbum has no inheritance association with item class.....or am i missing something

Reply:
i get it now